spec/filters/elasticsearch_spec.rb

```diff
@@ -712,9 +723,9 @@ def wait_receive_request
     end
 
     it "should read and send non-ascii query" do
-      expect(client).to receive(:search).with(
+      expect(client).to receive(:search).with({
```

Comment:
review note: rspec's message expectations differentiate between bare keyword arguments or a hash in Ruby 3.x (and so: JRuby 9.4.x from Logstash 8.10), so making this expect a hash fixes the build)